src/design-system/password-box/uncontrolled-password-box-input.component.tsx

```diff
@@ -68,6 +69,7 @@ export const UncontrolledPasswordInput = ({
               type={isPasswordVisible ? 'text' : 'password'}
               required={required}
               placeholder=""
+              autoFocus={autoFocus}
```

Comment:
I am ok with it as an interim solution but I have to point out that this technically may not work 100% of the time: https://blog.logrocket.com/how-to-autofocus-using-react-hooks/